felix/bpf/conntrack/bpf_scanner.go

```diff
@@ -145,17 +145,17 @@ func (s *BPFProgLivenessScanner) ensureBPFExpiryProgram() (*libbpf.Obj, error) {
 				continue
 			}
 
-			err := libbpf.CTCleanupSetGlobals(
+			err := libbpf.SetGlobalData(
```

Comment:
Could this be?

```suggestion
			err := &libbpf.CTCleanupGlobalData{...}.Set(m)
```

Reply:
Prefer doing it this way. Will update the PR shortly.